app/views/users/show.html.erb

```diff
@@ -4,7 +4,7 @@
     <div class='userinformation-inner'>
       <h1><%= @user.display_name %> <%= role_icons(@user) %></h1>
       <% if current_user and @user.id == current_user.id %>
-        <!-- Displaying user's own profile page to themself -->
+        <!-- Displaying user's own profile page -->
```

Comment:
`themself` is considered archaic. In order for it to be `user's own`, the observer has to be the user...

Reply:
The observer would be the user